services-camel-app/src/main/java/edu/unc/lib/boxc/services/camel/solrUpdate/SolrUpdateProcessor.java

```diff
@@ -113,6 +119,12 @@ public void process(Exchange exchange) throws Exception {
         }
     }
 
+    private static boolean actionDoesNotSupportTombstones(IndexingActionType actionType) {
```

Comment:
so only these three action types support tombstones?

Reply:
yeah, just indexing actions that delete records from solr. We don't support having solr records for tombstones at the moment